src/instruments/instruments.es6

```diff
@@ -13,14 +13,12 @@ function refresh_active_symbols() {
         .then(function(data) {
             local_storage.set('active_symbols', data.active_symbols);
             const active_symbols = [];
-            const active_markets = _(data.active_symbols).groupBy('market').map(function(symbols) {
-             
-                const filtered_symbols = filterRestrictedSymbols(symbols)
-  
-                const sym = _.head(filtered_symbols);
+            const active_markets = _(data.active_symbols).groupBy('market').map(function(activeSymbols) {
+               
+                const sym = _.head(activeSymbols);
      
                 const market = { name: sym.market, display_name: sym.market_display_name };
-                market.submarkets = _(filtered_symbols).groupBy('submarket').map(function(symbols) {
+                market.submarkets = _(activeSymbols).groupBy('submarket').map(function(symbols) {
```

Comment:
Rename `activeSymbols` to a snake_case variable name

Reply:
we have active_symbols already, so I renamed it to all_symbols